chart/values.schema.json

```diff
@@ -7601,6 +7601,14 @@
                     "additionalProperties": {
                         "type": "string"
                     }
+                },
+                "annotations": {
```

Comment:
You should also add something to `values.yaml` :) Makes it more easily discoverable.